src/main/game/entities/Entity.java

```diff
@@ -28,7 +28,7 @@ public Entity(Location location, EntityId entityId){
     public double getCurrentHealth(){ return health; }                              // Return entity health
     public Percentage getHealthPercentage(){return healthPercent; }                 // Return entity health pct.
     public void takeDamage(double damage){                                          // Take damage to health
-        this.health -= damage;
+        this.health -= (int)damage;
```

Comment:
health is an int in entity and stats. if you want it as a double i could work on changing it but int health agrees with the idea of hit points vs a health continuum